float64 x
float64 y
float64 z
float64[] orientation
float64 velocity
float64 acceleration
bool not_block
bool relative
string frame
---
string message

